web/src/beta/features/Dashboard/ContentsContainer/Members/ListItem.tsx

```diff
@@ -19,10 +70,50 @@ const ListItem: FC<{ member: TeamMember }> = ({ member }) => {
       </TypographyWrapper>
       <TypographyWrapper>
         <Typography size="body">
-          {member.role.charAt(0).toUpperCase() +
-            member.role.slice(1).toLowerCase()}
+          {isMemberRoleENG(memerRoleTranslation[member.role])
+            ? member.role.charAt(0).toUpperCase() +
+              member.role.slice(1).toLowerCase()
+            : memerRoleTranslation[member.role]}
```

Comment:
Why not use `{t(member.role)}` directly?

Reply:
If capitalize is necessary we can use `t(member.role).toLowerCase()` and give it `text-transform: capitalize;`

Reply:
Please update `UpdateRoleModal` roleOptions label using UPPERCASE directly so that we don't need to provide duplicate translations on i18n.